crates/apub/src/protocol/activities/private_message/delete.rs

```diff
@@ -11,6 +11,7 @@ use url::Url;
 #[serde(rename_all = "camelCase")]
 pub struct DeletePrivateMessage {
   pub(crate) actor: ObjectId<ApubPerson>,
+  #[serde(deserialize_with = "crate::deserialize_one")]
   pub(crate) to: [ObjectId<ApubPerson>; 1],
```

Comment:
Shouldn't this be added to every `to` field then? there are a lot more than just these ones on private message and reports.

Reply:
It's possible I missed some, but most were already changed in #1962

Reply:
Hrm... whats the difference between `Many`, and an array, the deserealize_on_or_many function looks almost duped in this one.

Reply:
it is almost the same, the difference is in return type, since these values are declared as 1-long arrays rather than Vecs

Reply:
Here it should be exactly one element, represented either as a single json value, or as a json array with one item. In #1962 it can be any number of elements, so either a json array, or a single json value (which gets converted to vec with one element). 

So yes the difference is the return type of the deserialize function (arbitrary size vec, or single element array). So its not possible to use the same function. And it looks like all activities have the annotation now, and can accept to/cc with single value for compatibility with more software.